src/coreclr/nativeaot/libunwind/src/Unwind-EHABI.cpp

```diff
@@ -91,9 +91,11 @@ _Unwind_Reason_Code ProcessDescriptors(
       case Descriptor::LU32:
         descriptor = getNextWord(descriptor, &length);
         descriptor = getNextWord(descriptor, &offset);
+        break;
```

Comment:
It would be nice to submit this patch to https://github.com/llvm/llvm-project/blob/main/libunwind/src/Unwind-EHABI.cpp as well (it is where this file came from).